.github/workflows/publish.yaml

```diff
@@ -0,0 +1,47 @@
+---
+name: Publish
+
+on:
+  release:
+    types:
+      - released
+      - prereleased
+  workflow_dispatch: {}
+
+permissions:
+  contents: read
+  id-token: write
+  packages: write
+
+jobs:
+  Sync:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Sync Repositories
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.GH_PERSONAL_ACCESS_TOKEN }}
+          script: |
+            const { data } = await github.rest.search.code({
+              q: 'Common Config org:beam-community path:.github/workflows/',
```

Comment:
@btkostner thoughts on how to makes this extensible to more than 1 org? 🤔

Reply:
No clue but there is probably some GH query for it. One thing to note is that the auth needs to have push permissions. So this dispatch works for beam org, all other orgs will just have to run via cron